### PR TITLE
0.2/124 fullscreen input tabs

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: src/components/Masthead.vue [hunk 1]
```diff
@@ -18,9 +18,9 @@
 
     <nav role="navigation" class="input-tab" v-if="($route.path==='/' || $route.path==='/tap' || $route.path==='/scan' || $route.path==='/upload') ? true : false">
         <ul>
-            <li><router-link v-bind:to="'/tap'">Type</router-link></li>
-            <li><router-link v-bind:to="'/scan'">Scan</router-link></li>
-            <li><router-link v-bind:to="'/upload'">Upload</router-link></li>
+            <router-link v-bind:to="'/tap'">Type</router-link>
+            <router-link v-bind:to="'/scan'">Scan</router-link>
+            <router-link v-bind:to="'/upload'">Upload</router-link>
         </ul>
```
Comment (katjam) [inline src/components/Masthead.vue]: Do you still want the ul?

**Review by katjam — CHANGES_REQUESTED** [review]
I think add the li back in or remove the ul

File: src/pages/Tap.vue [hunk 2]
```diff
@@ -46,7 +48,8 @@ export default {
         return {
             sponsorCode : "RVP",
             voucherCode : "",
-            vouchers : Store.vouchers
+            vouchers : Store.vouchers,
+            recVouchers : Store.recVouchers
```
Comment (charlesstrange2) [inline src/pages/Tap.vue:52]: Are we adding these back into /tap? i thought you moved them...

**Review by charlesstrange2 — APPROVED** [review]
:shipit:

**Pull request merged**